### ISSUE TITLE
On Android 5, about:crashcontent doesn't trigger the "close tab" page

### ISSUE THREAD

**Issue by lobontiumira** [mozilla-mobile/focus-android#5776]
## Steps to reproduce
1. Open a site, let it load fully, then type about:crashcontent.

### Expected behavior
The "Sorry. We're having problems with this tab.As a private browser, we never save and cannot restore this tab." messages are displayed."Send crash report to Mozilla" checkbox (checked by default)."Close tab" button. page is displayed.

![Screenshot_3](https://user-images.githubusercontent.com/40694158/140318774-533ef6f7-6b19-4a96-8c7c-0680e3bd79d7.jpg)

### Actual behavior
Only the "Sorry. Firefox Focus Beta had a problem and crashed." prompt is displayed.
![Screenshot_4](https://user-images.githubusercontent.com/40694158/140319425-1c6843f4-ff96-4ee2-8184-ae58324fa2c3.jpg)

### Device information
* Android device: Samsung Galaxy Tab A6 (Android 5.1.1), and Xiaomi mi4i (Android 5.0.2)
* Focus version: 95.0.0-beta.1
* Not reproducible on Sony Xperia Z5 Premium (Android 7.1.1)

[logcat.txt](https://github.com/mozilla-mobile/focus-android/files/7474958/logcat.txt)

**Comment by jonalmeida** [comment]
Looks like a colour resource isn't resolved during the `about:crashcontent` which is what crashes the app:

```
2021-11-04 14:57:34.153 16050-16050/? E/ExceptionHandler: Uncaught exception handled: 
    android.view.InflateException: Binary XML file line #20: Error inflating class <unknown>
        at android.view.LayoutInflater.createView(LayoutInflater.java:640)
        at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:55)
        at android.view.LayoutInflater.onCreateView(LayoutInflater.java:689)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:748)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:813)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:511)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:415)
        at org.mozilla.focus.fragment.CrashReporterFragment.onCreateView(CrashReporterFragment.kt:1)
        at androidx.fragment.app.Fragment.performCreateView(Fragment.java:4)
        at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:16)
        at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:24)
        at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:68)
        at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:10)
        at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:14)
        at androidx.fragment.app.FragmentActivity.onResume(FragmentActivity.java:5)
        at org.mozilla.focus.locale.LocaleAwareAppCompatActivity.onResume(LocaleAwareAppCompatActivity.java:1)
        at org.mozilla.focus.activity.MainActivity.onResume(MainActivity.kt:1)
        at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1285)
        at android.app.Activity.performResume(Activity.java:6730)
        at android.app.ActivityThread.performNewIntents(ActivityThread.java:3350)
        at android.app.ActivityThread.handleNewIntent(ActivityThread.java:3357)
        at android.app.ActivityThread.access$1800(ActivityThread.java:219)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1807)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:145)
        at android.app.ActivityThread.main(ActivityThread.java:6939)
        at java.lang.reflect.Method.invoke(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:372)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)
     Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Constructor.newInstance(Native Method)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
        at android.view.LayoutInflater.createView(LayoutInflater.java:614)
        at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:55) 
        at android.view.LayoutInflater.onCreateView(LayoutInflater.java:689) 
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:748) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:813) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:511) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:415) 
        at org.mozilla.focus.fragment.CrashReporterFragment.onCreateView(CrashReporterFragment.kt:1) 
        at androidx.fragment.app.Fragment.performCreateView(Fragment.java:4) 
        at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:16) 
        at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:24) 
        at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:68) 
        at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:10) 
        at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:14) 
        at androidx.fragment.app.FragmentActivity.onResume(FragmentActivity.java:5) 
        at org.mozilla.focus.locale.LocaleAwareAppCompatActivity.onResume(LocaleAwareAppCompatActivity.java:1) 
        at org.mozilla.focus.activity.MainActivity.onResume(MainActivity.kt:1) 
        at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1285) 
        at android.app.Activity.performResume(Activity.java:6730) 
        at android.app.ActivityThread.performNewIntents(ActivityThread.java:3350) 
        at android.app.ActivityThread.handleNewIntent(ActivityThread.java:3357) 
        at android.app.ActivityThread.access$1800(ActivityThread.java:219) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1807) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:145) 
        at android.app.ActivityThread.main(ActivityThread.java:6939) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:372) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199) 
     Caused by: android.content.res.Resources$NotFoundException: File res/yJ.xml from drawable resource ID #0x7f0800e8
        at android.content.res.Resources.loadDrawableForCookie(Resources.java:4061)
        at android.content.res.Resources.loadDrawable(Resources.java:3929)
        at android.content.res.Resources.loadDrawable(Resources.java:3779)
        at android.content.res.TypedArray.getDrawable(TypedArray.java:776)
        at android.view.View.<init>(View.java:4001)
        at android.view.View.<init>(View.java:3909)
        at android.view.View.<init>(View.java:3888)
        at java.lang.reflect.Constructor.newInstance(Native Method) 
        at java.lang.reflect.Constructor.newInstance(Constructor.java:288) 
        at android.view.LayoutInflater.createView(LayoutInflater.java:614) 
        at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:55) 
        at android.view.LayoutInflater.onCreateView(LayoutInflater.java:689) 
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:748) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:813) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:511) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:415) 
        at org.mozilla.focus.fragment.CrashReporterFragment.onCreateView(CrashReporterFragment.kt:1) 
        at androidx.fragment.app.Fragment.performCreateView(Fragment.java:4) 
        at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:16) 
        at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:24) 
        at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:68) 
        at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:10) 
        at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:14) 
        at androidx.fragment.app.FragmentActivity.onResume(FragmentActivity.java:5) 
        at org.mozilla.focus.locale.LocaleAwareAppCompatActivity.onResume(LocaleAwareAppCompatActivity.java:1) 
        at org.mozilla.focus.activity.MainActivity.onResume(MainActivity.kt:1) 
        at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1285) 
        at android.app.Activity.performResume(Activity.java:6730) 
        at android.app.ActivityThread.performNewIntents(ActivityThread.java:3350) 
        at android.app.ActivityThread.handleNewIntent(ActivityThread.java:3357) 
        at android.app.ActivityThread.access$1800(ActivityThread.java:219) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1807) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:145) 
        at android.app.ActivityThread.main(ActivityThread.java:6939) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:372) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199) 
     Caused by: android.content.res.Resources$NotFoundException: File res/uL.xml from color state list resource ID #0x7f080012
        at android.content.res.Resources.twLoadColorStateList(Resources.java:4179)
        at android.content.res.Resources.loadColorStateList(Resources.java:4117)
        at android.content.res.TypedArray.getColor(TypedArray.java:397)
        at android.graphics.drawable.VectorDrawable$VFullPath.updateStateFromTypedArray(VectorDrawable.java:1439)
        at android.graphics.drawable.VectorDrawable$VFullPath.inflate(VectorDrawable.java:1418)
        at android.graphics.drawable.VectorDrawable.inflateInternal(VectorDrawable.java:544)
        at android.graphics.drawable.VectorDrawable.inflate(VectorDrawable.java:460)
        at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:1192)
        at android.graphics.drawable.Drawable.createFromXml(Drawable.java:1086)
    	at android.content.res.Resources.loadDrawableForCoo
```

**Comment by lobontiumira** [comment]
Reproducible on RC 94.1.2 also, with Samsung Galaxy Tab A6 (Android 5.1.1).

**Comment by lobontiumira** [comment]
This is not fixed yet on Xiaomi mi4i (Android 5.0.2) device. After about:crashcontent, the previous page is displayed on top-half of the screen - please see the short video below:


https://user-images.githubusercontent.com/40694158/144605515-3a22cf96-3027-4028-a56c-7fb2b7441143.mp4

[logcat.txt](https://github.com/mozilla-mobile/focus-android/files/7649699/logcat.txt)


Everything works as expected on Samsung Galaxy Tab A6 (Android 5.1.1).

**Comment by lobontiumira** [comment]
I'll close this issue as fixed.
I've filed #5984 for the issue mentioned above, only reproducible on Xiaomi mi4i (Android 5.0.2).
